src/ucs/debug/log.c

```diff
@@ -164,30 +172,41 @@ static void ucs_log_file_rotate()
 
     for (idx = ucs_log_file_last_idx - 1; idx >= 0; --idx) {
         ucs_log_get_file_name(old_log_file_name,
-                              sizeof(old_log_file_name), idx);
+                              PATH_MAX, idx);
         ucs_log_get_file_name(new_log_file_name,
-                              sizeof(new_log_file_name), idx + 1);
+                              PATH_MAX, idx + 1);
 
         if (access(old_log_file_name, W_OK) != 0) {
-            ucs_fatal("unable to write to %s", old_log_file_name);
+            ucs_error("unable to write to %s", old_log_file_name);
```

Comment:
since we fatal anyways, maybe set `msg = "..."` then print with `ucs_fatal(".. %s.. ", msg);`

Reply:
or just replace ucs_error() by ucs_fatal()

Reply:
replace ucs_error() by ucs_fatal()

Reply:
replaced ucs_error() by ucs_fatal()